test/built-ins/Promise/all/iter-arg-is-false-reject.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2018 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-promise.all
+description: >
+  Reject when argument is `false`
+info: |
+    ...
+    Let iteratorRecord be GetIterator(iterable).
+    IfAbruptRejectPromise(iteratorRecord, promiseCapability).
+    ...
+
+    #sec-getiterator
+    GetIterator ( obj [ , hint [ , method ] ] )
+
+    ...
+    Let iterator be ? Call(method, obj).
+    If Type(iterator) is not Object, throw a TypeError exception.
+    ...
+features: [Symbol.iterator]
```

Comment:
some of these tests are not relying on the Symbol.iterator at all but just the _iterator_ argument used on Promise.all and .race. I believe these won't need this feature tag, unless Symbol.iterator is explicitly used.

This can be fixed in a follow up.

Reply:
They do implicitly, because the iterator protocol that the methods use looks up Symbol.iterator - anything with the iteration protocol relies on Symbol.iterator.